src/metadata.rs

```diff
@@ -77,4 +80,20 @@ impl SstFileMetaData {
             CStr::from_ptr(ptr).to_string_lossy().into_owned()
         }
     }
+
+    pub fn get_smallestkey(&self) -> &[u8] {
```

Comment:
Maybe it's better to use `get_smallest_key`.

Reply:
Both `smallestkey` and `smallest_key` are fine to me, I use `smallestkey` because RocksDB uses that too.